### ISSUE TITLE
Any recommendation loss scale for downstream task fine-tuing?

### ISSUE THREAD

**Issue by UniGen-X** [NVIDIA/Isaac-GR00T#165]
May I ask, for your approach, what would be a nice scale of the loss value for downstream task fine-tuning? (0,001, or 0.0001)? Thanks!

**Comment by shenyanqing1105** [comment]
> May I ask, for your approach, what would be a nice scale of the loss value for downstream task fine-tuning? (0,001, or 0.0001)? Thanks!

Same！！！

**Comment by xwjiang2010** [comment]
I think depend on the dataset.
For example, for finetuning on Libero Spatial, I got around 0.005, which already gives me above 90% success rate.
